src/app/projects/project/step.view.ts

```diff
@@ -79,17 +79,20 @@ export class StepModal implements VirtualDOM<'div'> {
                         stepId: this.step.id,
                         flowId: this.flowId,
                     })
-                    .pipe(raiseHTTPErrors()),
+                    .pipe(
+                        raiseHTTPErrors(),
+                        mergeMap((js) => from([js])),
+                        tap((v) => v),
+                    ),
                 vdomMap: (js: string) => {
+                    console.log('js :', typeof js)
```

Comment:
Remove this `console.log`

Reply:
Never mind, it is removed in a further commit

Reply:
it is removed in the commit [f518bd9](https://github.com/youwol/developer-portal/pull/21/commits/f518bd996c06a3a96d3f1bbae5bb1e695a8da594)